site/docs/enterprise/operator-openshift.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Hazelcast Jet Enteprise Operator on Red Hat Openshift
+description: Step-by-step guide to install Hazelcast Jet Enteprise Operator on Red Hat Openshift and deploying a sample job.
+---
+
+[OpenShift](https://www.openshift.com/learn/what-is-openshift) is a
+hybrid cloud, enterprise Kubernetes application platform from RedHat.
+[Red Hat Marketplace](https://marketplace.redhat.com/en-us), an open
+cloud marketplace that enables discovery and access to a certified
+software which is available to deploy on any Red Hat OpenShift cluster.
+
+[Hazelcast Jet
+Enterprise](https://marketplace.redhat.com/en-us/products/hazelcast-jet)
+is available to you on Red Hat Marketplace as a Certified Enterprise
+Ready Software.
+
+## Prerequisites
+
+- A running Openshift cluster either on-prem or hosted by RedHat.
```

Comment:
What do you mean by `hosted by RedHat`? Did you mean OpenShift Online? I guess our operator would not work there, because you don't have admin rights there.

Reply:
How about OpenShift Dedicated ?

Reply:
To be honest I don't know. I haven't tried OpenShift Dedicated.

Reply:
We can change the wording to include `OpenShift Cluster with administration rights` and remove the `on-prem and hosted` part